src/ensembl/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "copy-dotenv": "test ! -f .env && (cp .env.example .env; echo '.env file created') || true",
     "start": "npm install --no-save && npm run serve:dev",
-    "serve:dev": "npm run copy-dotenv && webpack-dev-server --config ./webpack/webpack.config.dev.js",
+    "serve:dev": "npm run copy-dotenv && webpack-dev-server --disable-host-check --config ./webpack/webpack.config.dev.js",
```

Comment:
The downside of removing this would be that we will be seeing many of those `invalid host/origin header` errors on IE11.

Reply:
I've left this disabled by default along with a comment, so that if someone comes across the same issue they only need to change the boolean value from false to true.